testplan/web_ui/testing/package.json

```diff
@@ -55,5 +57,6 @@
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
     ]
-  }
+  },
+  "proxy": "http://localhost:4000"
```

Comment:
Note: this setting allows us to start the UI in dev mode using `npm start` and have it communicate with the backend of a testplan, assuming the testplan is started in UI mode on port 4000 like `$ ./test_plan.py --ui 4000`. It's very useful as it allows you to iterate quickly on the frontend UI and test how it interacts with the real backend, instead of just loading the static fakeReport.js.

See https://create-react-app.dev/docs/proxying-api-requests-in-development for more detail.